media-sound/ncmpc/ncmpc-0.30.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit meson
+
+DESCRIPTION="A ncurses client for the Music Player Daemon (MPD)"
+HOMEPAGE="https://www.musicpd.org/clients/ncmpc/ https://github.com/MusicPlayerDaemon/ncmpc"
+SRC_URI="http://www.musicpd.org/download/${PN}/${PV%.*}/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~ppc ~ppc64 ~sparc ~x86"
+IUSE="artist-screen chat-screen doc +help-screen key-screen lirc lyrics-screen mouse nls outputs-screen search-screen song-screen"
+
+RDEPEND=">=dev-libs/glib-2.30:2
+	>=media-libs/libmpdclient-2.9
+	sys-libs/ncurses:0=[unicode]
+	lirc? ( app-misc/lirc )"
+DEPEND="${RDEPEND}
+	app-arch/xz-utils
+	virtual/pkgconfig"
+
+DOCS=( AUTHORS NEWS README.rst doc/config.sample doc/keys.sample )
+PATCHES=(
+	"${FILESDIR}/fix_enqueue_from_search.patch"
+	"${FILESDIR}/fix_gcc_warning.patch"
+	"${FILESDIR}/restore_progress_bar_in_colorless_mode.patch"
+)
+
+src_configure() {
+	local emesonargs=(
+		-Dcurses=auto
+		$(meson_use mouse)
+		-Dcolors=auto
```

Comment:
ditto

Reply:
After looking into meson.build, colors option is not read - instead curses_color is set based on what implementation of ncurses is found. So I removed it, so it can default to whatever it needs to be.